### ISSUE TITLE
Problem: task stderr is not captured after client script raises an exception

### ISSUE THREAD

**Issue by sevein** [artefactual/archivematica#1194]
How to reproduce:
1. Update `index_aip.index_aip()` so it raises an exception.
2. Start a transfer, store the AIP.
3. You should see something like the following: ![image](https://user-images.githubusercontent.com/606459/42414067-7079f1b2-81e2-11e8-9cd0-d43ce1499866.png)
4. Click on the gear icon.

The exit code is 179, this is expected (see #1190).
**However the stderr stream has not been recorded. It's not in the database.**

![image](https://user-images.githubusercontent.com/606459/42414085-c87a0104-81e2-11e8-9fcd-b8b57a9547f5.png)

I would expect to see the message of my exception.

This was originally found by @jhsimpson - see https://github.com/artefactual/archivematica/pull/1190#pullrequestreview-135179192.

**Comment by nickwilkinson** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Jisc RDSS](https://trello.com/c/onaPPfEg/3-jisc-rdss)

**Comment by joel-simpson** [comment]
I attempted to test this in our Jisc QA environment. To reproduce, I started with several SIPs in the Ingest stage that were awaiting user input (```Job Reminder: add metadata if desired```). I then stopped the ElasticSearch service (container), and provided the user input necessary to continue the workflow.

The Index AIP job fails as expected. 
![screenshot from 2018-07-23 18-03-45](https://user-images.githubusercontent.com/17187309/43105491-18560c14-8ea3-11e8-8aa9-8e9c295c225e.png)

BUT the stderr is still not present when I click on the gear icon to see the task details: 
![screenshot from 2018-07-23 18-03-29](https://user-images.githubusercontent.com/17187309/43105516-2a52bc82-8ea3-11e8-99a0-209b39c074e0.png)

I didn't quite follow in the instructions @sevein suggested to reproduce, but I think the way I did it is still valid to test this? (also we confirmed in the VM that we deployed with the latest branch of qa/1.x; and I also restarted all services to make sure this had happened)
